### DISCUSSION TITLE
face detection [AI-DECK]

### DISCUSSION THREAD

**Discussion by coo122567** [bitcraze/discussions#452] [Q&A]
Hello everyone,
May I ask what is the problem in the picture below, I can't use face detection?
![image](https://user-images.githubusercontent.com/118537645/205687127-7e7e6677-a91c-4dc6-9bf4-97223bd26509.png)

**Comment by knmcguire** [comment]
There is an issue with the latest GAP sdk for the facedetection example: https://github.com/bitcraze/aideck-gap8-examples/issues/92

We haven't gotten around to fix it but you can try downgrading your GAP sdk back to before 4.0
